index.bs

```diff
@@ -4937,6 +5031,50 @@ side=], or to terminate or error the stream.
  1. Return [=this=].\[[writable]].
 </div>
 
+<h4 id="ts-transfer">Transfer via postMessage()</h4>
+
+<dl class="domintro">
+ <dt><code>worker.postMessage(ts, [ts]);</code>
+ <dd>
+  <p>Sends a TransformStream to another frame, window or worker.
+
+  <p>The transferred stream can be used exactly like the original. The original will become locked
+  and no longer directly usable.
+ </dd>
+</dl>
+
+<div algorithm="TransformStream transfer steps">
+{{TransformStream}} objects are [=transferable objects=]. Their [=transfer steps=], given |value|
+and |dataHolder|, are:
+
+<!-- Find a way to linkify ReadableStream transfer steps and WritableStream transfer steps -->
+
+ 1. If ! [$IsReadableStreamLocked$](|value|.\[[readable]]) is true, throw a "{{DataCloneError}}"
+    {{DOMException}}.
+ 1. If ! [$IsWritableStreamLocked$](|value|.\[[writable]]) is true, throw a "{{DataCloneError}}"
+    {{DOMException}}.
+ 1. Perform {{ReadableStream}} transfer steps with |value|.\[[readable]] and |readableDataHolder|.
```

Comment:
I think you can just do the same thing you did for MessagePort, passing them into StructuredSerializeWithTransfer.

Reply:
Done.